.github/workflows/ci.yml

```diff
@@ -36,7 +42,7 @@ jobs:
         with:
           channel: "stable"
       - run: dart pub get
-      - run: dart test
+      - run: dart test --platform vm
 
   # TODO: Enable package:media_kit tests on macOS
```

Comment:
Is it really necessary to test `media_kit` on all platforms? Knowing that it's ffi, there's no risk of the tests failing on one platform when they succeeded on others. Maybe keeping the tests on linux is enough.

Reply:
Linking is already tested in the rest of the CI. And lib builds are tested manually.

Reply:
Yeah! Don't feel I'm asking to run the tests on macOS. GNU/Linux unit-tests are surely enough & sufficient.

Just wanted to run these on Windows... Windows is kind of different.